Remove fixed width for login-logo

pascalwengerter opened the pull request:
Fixes cases like this

![Screenshot_2021-04-09 accessDenied - ownCloud](https://user-images.githubusercontent.com/16822008/114190975-d3311580-9943-11eb-9911-2477a5d68cb9.png)

kulmann reviewed (commented):
What if a logo is much taller than wide? As an alternative we could set a `max-height` and `max-width` of 200px each. That would still keep the correct aspect ratio while not shrinking to tiny sizes.

pascalwengerter commented:
> What if a logo is much taller than wide? As an alternative we could set a `max-height` and `max-width` of 200px each. That would still keep the correct aspect ratio while not shrinking to tiny sizes.

Wasn't aware using the combination of max-width & max-height keeps the aspect ratio in a proper way, nice! Implemented and updated the PR accordingly.

kulmann reviewed (approved):
👍
